Sources/Honeycomb/Networking/NetworkInstrumentation.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+import OpenTelemetryApi
+import SwiftUI
+import UIKit
+
+private let urlSessionInstrumentationName = "@honeycombio/instrumentation-urlsession"
+
+/// Creates a span with attributes for the given http request.
+internal func createSpan(from request: URLRequest) -> any Span {
+    let tracer = OpenTelemetry.instance.tracerProvider.get(
+        instrumentationName: urlSessionInstrumentationName,
+        instrumentationVersion: honeycombLibraryVersion
+    )
+
+    var builder = tracer.spanBuilder(spanName: request.httpMethod ?? "UNKNOWN")
+    builder.setSpanKind(spanKind: SpanKind.client)
+    if let method = request.httpMethod {
+        builder = builder.setAttribute(key: "http.request.method", value: method)
```

Comment:
can/should we use the semantic attributes from the `OpenTelemetryApi` module?

Reply:
Done.

Thanks, this is much nicer. Interestingly, I had to switch to setting them on the `Span`, because that setter wasn't present on the `SpanBuilder`. I think maybe that's an oversight we could contribute upstream? I'll file a ticket.